documentation/cookbook/working-with-existing-databases.markdown

```diff
@@ -44,15 +45,13 @@ Because Propel has both the ability to create XML schema files based on existing
 To do this you would simply:
 
  1. Follow the steps above to create the `schema.xml` file from existing db.
- 2. Then you would change the target database type and specify connection URL for new database in the project's configuration file (see [configuration](/documentation/10-configuration.html) document)
-
- 3. And then run the `sql:build` task to generate the new DDL:
+ 1. Then you would change the target database type and specify connection URL for new database in the project's configuration file (see [configuration](/documentation/10-configuration.html) document)
```

Comment:
Ah yes! Actually this is a known issue of Redcarpet (the Markdown parser we are relying on). Could you simply remove this change please ? We will handle that in another commit.